COVIDSafepassage/passsystem/tests.py

```diff
@@ -1,3 +1,163 @@
 from django.test import TestCase
 
-# Create your tests here.
+import factory

```

Comment:
is this a part of django or a separate module?

Reply:
its a separate module
pip install factory-boy

Reply:
please add to requirements.txt